filters/EigenvaluesFilter.cpp

```diff
@@ -65,6 +65,9 @@ void EigenvaluesFilter::addArgs(ProgramArgs& args)
 {
     args.add("knn", "k-Nearest neighbors", m_knn, 8);
     args.add("normalize", "Normalize eigenvalues?", m_normalize, false);
+    args.add("stride", "Compute features on strided neighbors", m_stride, size_t(1));
```

Comment:
There are some checks/feedback that could be provided WRT various options being incompatible.

Reply:
let me know if latest commit bring some more clarity